test/ask/survey_test.exs

```diff
@@ -207,4 +207,44 @@ defmodule Ask.SurveyTest do
       assert created_survey.panel_survey_id == panel_survey.id
     end
   end
+
+  describe "success_rate_history" do
```

Comment:
I think these tests would be more complete if they included examples where the values weren't always `0.0`. Do you agree?

Reply:
Yes, but this is already tested at a higher level (the controller), so I'm a little lazy to generate actual data for these tests. In fact, I only needed them to write the model method, and maybe we don't need these tests anymore?

Reply:
I'm not sure. I wouldn't remove them. I think I'd prefer to have at least a couple of simpler and more descriptive (unit) tests here. These tests don't pretend too much, but they explain part of the new behavior. The tests in `RespondentControllerTest` are so many (4893 lines) and so complex. I'd love to have some smaller, simpler, and more descriptive tests here rather than there, not only for coverage reasons (I guess the coverage is OK) but for readability purposes too.

I think we're OK regarding the scope of this issue. Maybe in the future, we can refactor/rethink our test suite regarding its coding style/design?